Ecommerce/index.html

```diff
@@ -1,13 +1,32 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>E-commerce</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <!-- Start coding from the lines below for ECOMMERCE TASK -->
-    <!-- to import image from public folder, do it like this = <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+
+<!DOCTYPE html>
+<html><head><title>Task1</title></head>
+<body>
+<h1>Welcome to My Shop</h1>
+<img src="shp.png">
+<ul>
+<li><a href="#">About Us</a></li>
+<li><a href="#">Products</a></li>

```

Comment:
The 'action' attribute of the form should specify the URL to which the form data will be submitted.